style.css

```diff
@@ -1,15 +1,14 @@
-@import "/home/jle/.cache/squash/colors.css";
```

Comment:
I don't support this change as this decrease the modularity of the system. It is intended to be used with my theming application, `squash`, where colors are sourced from a common location on the system so they don't have to be duplicated all over the filesystem.

However, I note that not everyone is going to be using this, so I'm willing to consider a work around. What if we included two imports, as such:

```css
@import "./colors.css";
@import "/home/jle/.cache/squash/colors.css";
```
If the cached colors don't exist, we could use a set of included defaults. Otherwise, we use the `squash` specified colors.

I'm assuming this would work, feel free to correct me if I'm wrong.